code/__DEFINES/job.dm

```diff
@@ -51,7 +51,7 @@
 #define JOB_CLOWN				(1<<11)
 #define JOB_MIME				(1<<12)
 #define JOB_CIVILIAN			(1<<13)
-
+#define JOB_EXPLORER			(1<<14)
```

Comment:
Tested locally, this flag can fit into the DB table

Reply:
This should never end up in DB table, since players can't set a preference for it - it won't even show in the prefs screen.